### ISSUE TITLE
Super+/ Binding listed twice, in two sections, with different descriptions

### ISSUE THREAD

**Issue by ids1024** [pop-os/shell-shortcuts#34]
The "Navigate Applications and Windows" section lists the `Super+/` binding as "Launch and Switch Applications". It is also listed under "Launcher Shortcuts", as "Activate Launcher".

1. Do we want it listed in two places? It makes sense in both, but the duplication seems weird.
2. If we remove one, which should be kept?
3. In either case, presumably it should be described only one way. What should it be called?

**Comment by mmstick** [comment]
That'd be a question for @maria-komarova

**Comment by maria-komarova** [comment]
Yeah, that's somewhat weird. I would still leave that in both places considering people might be looking in different places for the and since we have Launcher shortcuts as its own category. 
In terms of the description, maybe something like Activate Launcher to Open and Switch Applications? If that is not too long.

**Comment by ids1024** [comment]
A bit verbose, but seems to fit with others:

![Screenshot from 2020-09-02 10-11-06](https://user-images.githubusercontent.com/2263150/92014783-a4472700-ed04-11ea-9dd4-304ad61386bc.png)

I suppose we also want to use the same description in both gnome-control-center and shell-shortcuts.

I'm not really sure whether or not the verbosity of this description succeeds in making it clear what the shortcut does. Hm...

**Comment by maria-komarova** [comment]
Yeah... I am not sure either. After pondering that more we should probably stick with "Launch and switch applications" for all three places. Somewhat less verbose but still gives one the end result.
